src/Microsoft.AspNetCore.Sockets/HttpConnectionDispatcher.cs

```diff
@@ -49,128 +45,141 @@ public HttpConnectionDispatcher(ConnectionManager manager, ILoggerFactory logger
 
         private async Task ExecuteEndpointAsync(string path, HttpContext context, EndPoint endpoint)
         {
-            // Server sent events transport
-            if (context.Request.Path.StartsWithSegments(path + "/sse"))
```

Comment:
Also Long Polling having such a complicated execution model. I'm not sure I even fully understand it yet, it's pretty sketchy right now.